### PR TITLE
Test sentry-resque without Rails, fix `constantize` issue

### PR DESCRIPTION
**Pull request by dentarg**
Fixes https://github.com/getsentry/sentry-ruby/pull/2243

### CODE REVIEW

File: sentry-resque/Gemfile [hunk 1]
```diff
@@ -4,12 +4,6 @@ git_source(:github) { |name| "https://github.com/#{name}.git" }
 # Specify your gem's dependencies in sentry-ruby.gemspec
 gemspec
 gem "sentry-ruby", path: "../sentry-ruby"
-gem "sentry-rails", path: "../sentry-rails"
-
-gem "rails"
-
-# For https://github.com/ruby/psych/issues/655
-gem "psych", "5.1.0"
```
Comment (dentarg) [inline sentry-resque/Gemfile:12]: This makes tests [pass](https://github.com/getsentry/sentry-ruby/actions/runs/7942303945/job/21685467789) on Ruby 2.4 ([fail log before this change](https://github.com/getsentry/sentry-ruby/actions/runs/7942214345/job/21685304613#step:7:55))

**Review by st0012 — COMMENTED** [review]
Thanks for the fix 👍 
Would you mind updating the changelog too?

**Comment by dentarg** [comment]
What do you want it to say?

**Comment by st0012** [comment]
Something like:

```md
- Fix undefined method `constantize' issue in sentry-resque ([#2248](https://github.com/getsentry/sentry-ruby/pull/2248))
```

**Comment by dentarg** [comment]
Added to the changelog ✔️

**Review by st0012 — APPROVED** [review]
Thanks 👍

**Pull request merged**